core/cache/Readme.md

```diff
@@ -56,6 +56,32 @@ response, err := apiclient.Cache.Get(requestContext, u.String(), loadData)
 ## Cache backends
 
 Currently there are the following backends available:
-* inMemoryCache (caches in memory - and therefore is a very fast cache)
-* fileBackend (caches in filesystem )
-* nullBackend (caches nothing)
\ No newline at end of file
+
+### inMemoryCache
```

Comment:
@danielpoe, @bastianccm 

I started some basic documentation and i am open for input.